Assignment/Program.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
-   }
+﻿Console.Write ("Enter a value : ");
+int a =int.Parse(Console.ReadLine ());

```

Comment:
What if a and b are null ?

Reply:
When a and b will be null, since "int" is a value type.
Can you please explain with an example?